src/common/darktable.c

```diff
@@ -2040,9 +2040,9 @@ void dt_show_times_f(const dt_times_t *start,
 
 int dt_worker_threads()
 {
-  const size_t threads = dt_get_num_threads();
-  const size_t mem = _get_total_memory();
-  const int wthreads = (mem >= (8lu << 20) && threads >= 4) ? 4 : MIN(2, threads);
+  const int threads = dt_get_num_threads();
+  const int gbytes = (int)(_get_total_memory() / (1lu << 20));
```

Comment:
1lu << 20 or (1024 * 1024)
No big deal, but sometime I'm wondering how hard we want a code to be readable :)
I would even have a named number
`const int onegb = 1024 * 1024;`
And use it.

Reply:
And I understand that this is not your code, you have just copy/pasted it.